samples/tutorials/Data passing in python components.ipynb

```diff
@@ -5,7 +5,7 @@
    "metadata": {},
```

Comment:
<p>FYI, from now on we can use <code>XXXX.pipelines.googleusercontent.com</code> for better branding. Or if we prefer not to confuse users we shall do <code>XXXX.{pipelines|notebooks}.googleusercontent.com</code>   <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/2868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Fixed</p>